config.json

```diff
@@ -423,6 +423,14 @@
         "pattern matching",
         "stacks"
       ]
+    },
+    {
+      "slug":"pangram",
```

Comment:
This needs a couple more fields (the `uuid`, which configlet can generate for you, `core` and `unlocked_by` (just leave the values the same default as the others.

Reply:
What command do I run on the configlet to get it to add the `uuid`? I've tried `fmt` and `lint` and neither seems to be doing it.

Reply:
I should clarify this - `configlet uuid` will generate the uuid string for you, but you'll need to edit the json manually